.github/workflows/build.yml

```diff
@@ -2,52 +2,18 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
-  workflow_dispatch:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
Remove on `pull_request`. We don't want to build and publish docker images on PR